include/inviwo/core/util/imageramutils.h

```diff
@@ -98,6 +99,12 @@ void forEachPixelParallel(const LayerRAM& layer, C callback, size_t jobs = 0) {
     forEachPixelParallel(layer.getDimensions(), callback, jobs);
 }
 
+IVW_CORE_API void flipLayerVertical(std::shared_ptr<Layer> layer);
```

Comment:
Generally, we do not want to take shared_ptr as input. Might want to change that for the foreach layer?

Reply:
Valid point. Also needs documentation in `foreachLayer`